src/containers/app/App.jsx

```diff
@@ -36,6 +41,18 @@ class App extends Component {
     })
   }
 
+  componentDidMount = () => {
+    window.addEventListener('resize', this.updateDimensions);
```

Comment:
Shouldn't update on resize, our concern is metamask not the actual size of the screen

Reply:
The initial issue was mobile devices & if a mobile screen is detected then do not display the bridge form. Because of the refactor in App.js its not easy to use any of the libraries for responsive screen sizes. @ansermino